ikarus/python/test/trusstest.py

```diff
@@ -0,0 +1,131 @@
+# SPDX-FileCopyrightText: 2021-2024 The Ikarus Developers mueller@ibb.uni-stuttgart.de
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+
+import debug_info
+
+debug_info.setDebugFlags()
+
+import ikarus as iks
+from ikarus import finite_elements, assembler
+import numpy as np
+import scipy as sp
+
+import dune.foamgrid
+import dune.grid
+import dune.functions
+
+
+def trussTest(worldDim):
+    assert worldDim == 2 or worldDim == 3
+    gridDim = 1
+    filename = f"auxiliaryfiles/truss{worldDim}d.msh"
```

Comment:
I just remembered that there is a nice way of accesing files independent of your working directory
```py
import os
dir = os.path.dirname(__file__)) # this is always the absolute path to the current python file
filename = os.path.join(dir, f"auxiliaryfiles/truss{worldDim}d.msh")
```

Reply:
Is this necessary? As now everything works? Do you know a case where it will fail?

Reply:
I think it would be nice, so I am always running the python scripts form the top-level directory via VS Code, and for this case the filepath is wrong atm. 
With my suggestion it won't matter from where you run the script

Reply:
Okay done